### DISCUSSION TITLE
KeyError: 'unit_id'

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#165] [Q&A] [answered]
Hello!

I have the following issue and don't know how to solve it. I run minian and after the pipline is done I use the cnmfviewer to manually inspect the extracted ROIs. Once I'm done I run the two cell that assigns the corrected labels and then the one that saves the values but I then get the following error message:

![Capture](https://user-images.githubusercontent.com/11939642/142064891-4318e536-536b-4b6d-8ee8-0d26ad6db79b.PNG)
:

I'm really desperate about this issue right now. Does anybody know how to solve it?

Thanks,
Thomas

**Comment by phildong** [comment]
not really sure what's going on. Could you post more detail on how to reproduce this? i.e., how does your notebook deviate from the default one? do you already have some data under the save / intermediate path?

**Comment by tom221085** [comment, marked as answer]
Hi phildong,
thank you for the answer! For some unknown reason the issue doesn't appear anymore. But I will post a more specific reproduction in case it happens again.
